### ISSUE TITLE
pockyt reg hanging

### ISSUE THREAD

**Issue by jamespo** [achembarpu/pockyt#20]
This is with v1.4.2 in a python 3.6 virtualenv on ubuntu 18.04 aarch64.

Sits at "Note: During the registration process, pockyt will attempt to open the required links in your default browser. If any errors occur, you can use the printed links instead."

This is in the console, no X session.

**Comment by achembarpu** [comment]
I haven't tested this in a console-only scenario, but I would hope that it print an error instead of hanging. Do you have any more info to share to debug this? Thanks!

**Comment by jamespo** [comment]
I'll have a look... what other similar tools do is print the URL they try to launch so you can copy/paste it into a browser, so if I get a chance I'll see how this can be implemented/

**Comment by xxKeith20xx** [comment]
Hello! I am getting the hang as well:
```
> pockyt reg
Note: During the registration process, pockyt will attempt to open the required links in your default browser. If any errors occur, you can use the printed links instead.
```

debian testing, installed with *pipx*

**Comment by achembarpu** [comment]
Hey all, I've fixed this in v1.4.4 with the flag `--no-browser`.
Do re-open the issue if it doesn't work for you. Thanks!
